packages/calling/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -114,12 +148,13 @@ describe('CallingClient Tests', () => {
     it('Verify invalid service indicator, empty domain', async () => {
       /* eslint-disable @typescript-eslint/no-explicit-any */
       const serviceDataObj: any = {indicator: 'test', domain: ''};
-
-      expect(() => {
-        callingClient = createClient(webex, {serviceData: serviceDataObj});
-      }).toThrow(Error);
-      expect(callingClient).toBe(undefined);
-      expect(createRegistrationSpy).not.toHaveBeenCalled();
+      try {
+        callingClient = await createClient(webex, {serviceData: serviceDataObj});
```

Comment:
What if the createClient doesn't throw any error? The test case would still pass.
Suggestion:
* to use [expect.assertions(1)](https://jestjs.io/docs/expect#expectassertionsnumber)
* use `.toThrow()`

Reply:
Updated code with `expect.assertions`.